Stepper: add Getting Started (#7318)
DevExpress/devextreme-documentation: concepts/05 UI Components/Stepper/05 Getting Started with Stepper/20 Configure Selection.md

> Stepper offers multiple properties that you can use to configure selection at runtime:

Reviewer: ```suggestion
The following Stepper properties allow you to configure selection at runtime:
```

            const newItem = e.addedItems[0];
            const items = e.component.option('items');
            const newIndex = items.findIndex((item) => newItem.label === item.label);
            e.component.option(`items[${newIndex - 1}].disabled`, true);
        }
    }

##### Vue

    <!-- tab: App.vue -->
    <script setup lang="ts">
        // ...
        const onSelectionChanged = (e: DxStepperTypes.SelectionChangedEvent) => {
            const newItem = e.addedItems[0];
            const items = e.component.option('items');
            const newIndex = items.findIndex((item) => newItem.label === item.label);
            e.component.option(`items[${newIndex - 1}].disabled`, true);
        };
    </script>
    <template>
        <DxStepper @selection-changed="onSelectionChanged">
            <!-- ... -->
        </DxStepper>
    </template>

##### React

    <!-- tab: App.tsx -->
    const onSelectionChanged = (e: StepperTypes.SelectionChangedEvent) => {
        const newItem = e.addedItems[0];
        const items = e.component.option('items');
        const newIndex = items.findIndex((item: StepperTypes.Item) => newItem.label === item.label);

>         e.component.option(`items[${newIndex - 1}].disabled`, true);

Reviewer 2: That is very unnatural way in React to update component's state, React expects declarative style: we should just change step state. It is not so clear how to support it with proposed structure of items

My suggestion:
1. Let's use array, which may have the same format with empty objects
2. For template we can use named template and add it to configuration of corresponding step
3. onSelectionChanged updates steps state instead of using e.component.option directly

I'm not equally experienced in other frameworks, but sure, that angular developer also expect class field to be updated, and vue developers expect using ref
